server/src/main/java/org/opensearch/common/cache/stats/DefaultCacheStatsHolder.java

```diff
@@ -208,15 +210,14 @@ private ImmutableCacheStats removeDimensionsHelper(List<String> dimensionValues,
         return statsToDecrement;
     }
 
-    // pkg-private for testing
-    Node getStatsRoot() {
+    public Node getStatsRoot() {
         return statsRoot;
     }
```

Comment:
nit: Why does this need to be public? I don't any other usage throughout this PR